src/makeRobotAccountant.js

```diff
@@ -6,7 +6,27 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  function getSum(number) {
+    count++;
+
+    if (count <= 3) {
+      return function(nextNumber) {
+        return number + nextNumber;
+      };
+    } else {
+      return function(nextNumber) {
+        if (count % 2 === 0) {
+          return 'Bzzz... Error!';
+        } else {
+          return number + nextNumber;
+        }
+      };
+    }
```

Comment:
create `isBroken` variable, which includes both problems of the robot and return the appropriate values based on this